### PR TITLE
Pull request

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: 0x000.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# 0x000 :octocat:
+ Hi, this is [0x000](https://github.com/0x000)
```
Comment (muan) [inline 0x000.md]: :wave: is that an extra space before `Hi`?

**Comment by ghost** [comment]
Thank you! :relaxed:

**Comment by alfredxing** [comment]
:100: 
Awesome! Your page is at http://muan.co/hello-world/0x000.md :tada:

File: 0x000.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# 0x000 :octocat:
```
Comment (muan) [inline 0x000.md:1]: If you add 

```
---
---
```

at the beginning of this file, it will be turned into html also and the markdown will be rendered!

**Comment by alfredxing** [comment]
@muan Right :laughing:. Sorry for not catching that (I was wondering why it was not generating the HTML...)!

File: 0x000.md (hunk 2 above)
Comment (alfredxing) [inline 0x000.md:1]: Right :laughing:. Sorry for not catching that (I was wondering why it was not generating the HTML...)!

Comment (muan) [inline 0x000.md:1]: No worries, I also missed it when I first reviewed it. :)

**Comment by Alexanamoon** [comment]
hello!
I would like to address to you, as IT specialist.
I developed the concept of a startup employment that would allow travelers, migrants, refugees and all other people who are interested in working abroad, to freely seek work anywhere in the world (not limited by language barriers). 
Unfortunately I do not have nothing in common with programming and application development. Accordingly, I need a developer. I would be very happy if you would consider my proposal. 
P.S. I live in Ukraine (www.facebook.com/vadim00)
Thank U.

File: 0x000.md (hunk 2 above)
Comment (ghost) [inline 0x000.md:1]: Adding the dashes at the beginning of the file in that way causes this:

```
---
---
```

immagine

Comment (muan) [inline 0x000.md:1]: Are you familiar with [Jekyll](https://help.github.com/articles/using-jekyll-with-pages/)? When it tries to generate the html from this markdown file, the content within `---` will be parsed out as [front matter](http://jekyllrb.com/docs/frontmatter/).

**Pull request merged**